test/handler.test.js

```diff
@@ -231,6 +231,51 @@ describe('strong-error-handler', function() {
         });
       });
 
+    it('should allow setting safe fields when status=5xx', function(done) {
+      // Mock an error reported by fs.readFile
```

Comment:
The comment is not accurate, the error object does not resemble errors reported by `fs.readFile`. Please remove this line.